spec/cypress/support/step_definitions/factory-bot.js

```diff
@@ -51,6 +51,24 @@ const login = (traits, args) => {
     });
 };
 
+const loginAdmin = (args) => {
+  const factoryArgs = ["create", "user", "admin"];
+
+  if (args) {
+    factoryArgs.push(parseArgs(args));
+  }
+
+  cy.appFactories([factoryArgs])
+    .as("userData")
+    .then(([user]) => {
+      cy.visit(`/users/confirm-sign-in?login_token=${user.login_token}`);
+    });
+
+  cy.get('[action="/users/sign-in-with-token"] [name="commit"]').click();
+};
+
+Given("I am admin and logged in", () => loginAdmin());
+Given("I am admin and logged in with {}", (args) => loginAdmin(args));
```

Comment:
I'd prefer it if these were just a special case in the existing step definitions (and that would be backwards compatible, too)

Reply:
I made admin a special case in our definitions, looks a bit better to me :)